core/clipboard/block_paster.ts

```diff
@@ -22,15 +22,17 @@ export class BlockPaster implements IPaster<BlockCopyData, BlockSvg> {
   ): BlockSvg | null {
     if (!workspace.isCapacityAvailable(copyData.typeCounts!)) return null;
 
-    const state = copyData.saveInfo as State;
```

Comment:
Curiosity: are you removing this just because it isn't a very useful temporary binding, or is there a type-checking reason?

Reply:
This used to be typed as `Object | Element` because workspace comments and blocks were returning the same "interface" so I needed to cast it to `State`. But since they are now using separate interfaces, there's no need to cast =)